src/app/components/login/login.module.ts

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
Commented code should not be commited. 
You should finish work or don't send it at all